guides/source/getting_started.md

```diff
@@ -1991,6 +1991,191 @@ the `app/views/comments` directory.
 The `@article` object is available to any partials rendered in the view because
 we defined it as an instance variable.
 
+
+### Using Concerns
+
+Concerns are a way to make large controllers or models easier to understand and manage. This also has the advantage of reusability when multiple models (or controllers) share the same concerns. Concerns are implemented using modules that contain methods representing a well-defined slice of the functionality that a model or controller is responsible for. In other languages, modules are often known as mixins.
+
+You can use concerns in your controller or model the same way you would use any module. When you first created your app with `rails new blog`, two folders were created within `app/` along with the rest:
+
+ ```
+ app/controllers/concerns
+ app/models/concerns
+ ```
+
+A given blog article might have various statuses - for instance, it might be visible to everyone (i.e. `public`), or only visible to the author (i.e. `private`). It may also be hidden to all but still retrievable (i.e. `archived`). Comments may similarly be hidden or visible. This could be represented using a `status` column in each model.
+
+Within the `article` model, after running a migration to add a `status` column, you might add:
+
+```ruby
+class Article < ApplicationRecord
+  has_many :comments
+  validates :title, presence: true,
+                    length: { minimum: 5 }
```

Comment:
@mcgregordan I see what you're saying but I don't think I should change this, because this is copied from the earlier model code snippets in the Getting Started guide. Someone following the guide would get confused if we're talking about the same Article and Comment models that we were discussing in the previous sections of the guide, but then suddenly they were missing their validations and such.